package.json

```diff
@@ -14,6 +14,7 @@
     "test:config": "jest src/config/__tests__/ --testPathIgnorePatterns",
     "integration-test": "start-server-and-test start http://localhost:3000 'cypress run'",
     "lint": "eslint 'src/**/*.{js,jsx,ts,tsx}'",
+    "lint:style": "npx stylelint '**/*.{js,jsx,ts,tsx}'",
```

Comment:
II think we only need to lint `tsx` ?

Reply:
Yeah, we don't need js and jsx
but, I think we need ts either because styled-components can be in ts file also( export and const only )

Reply:
Yeah. You're right.